test/e2e/monitor_resources.go

```diff
@@ -40,8 +40,8 @@ var allowedUsage = resourceUsagePerContainer{
 	},
 	"/kubelet": &containerResourceUsage{
 		CPUUsageInCores:         0.1,
-		MemoryUsageInBytes:      150000000,
-		MemoryWorkingSetInBytes: 150000000,
+		MemoryUsageInBytes:      1000000000,
```

Comment:
Did we decided that we want to do it?
I thought that we want to change set the limits appropriately according to what was written in:  https://github.com/kubernetes/kubernetes/issues/17774#issuecomment-159705367

So to be honest - I would prefer to hold on with this change.